dotCMS/src/integration-test/java/com/dotcms/datagen/TemplateDataGen.java

```diff
@@ -167,6 +169,16 @@ public TemplateDataGen withContainer(final String containerId, final String UUID
         return this;
     }
 
+    public TemplateDataGen withContainer(final Container container, final String UUID) {
```

Comment:
Isn't a more descriptive (business logic) name instead of "UUID" ?

Reply:
Yes, but it already is named UUID everywhere